src/storage/src/compaction/task.rs

```diff
@@ -55,6 +158,36 @@ pub struct CompactionOutput {
     pub(crate) inputs: Vec<FileHandle>,
 }
 
+impl CompactionOutput {
+    async fn run(
+        &self,
+        schema: RegionSchemaRef,
+        sst_layer: AccessLayerRef,
+        object_store: ObjectStore,
+    ) -> Result<FileMeta> {
+        let reader = build_sst_reader(
+            schema,
+            sst_layer,
+            &self.inputs,
+            self.bucket_bound,
+            self.bucket_bound + self.bucket,
+        )
+        .await?;
+        let output_file_name = format!("{}.parquet", Uuid::new_v4().hyphenated());
+        let opts = WriteOptions {};
+        let SstInfo { time_range } =
+            ParquetWriter::new(&output_file_name, Source::Reader(reader), object_store)
+                .write_sst(&opts)
+                .await?;
+
+        Ok(FileMeta {
+            file_name: output_file_name,
```

Comment:
Consider just holding uuid in `FileMeta`. Or define a type like `struct FileName(Uuid)` or `struct FileId(Uuid)`. We could provide a method to generate the `output_file_name`.

Reply:
I create a future work item in https://github.com/GreptimeTeam/greptimedb/issues/930.